### DISCUSSION TITLE
Tailwind styles not being updated after save with Next Js.

### DISCUSSION THREAD

**Discussion by afkcodes** [tailwindlabs/tailwindcss#6265] [Help]
**What version of Tailwind CSS are you using?**

For example: v2.2.19

**What build tool (or framework if it abstracts the build tool) are you using?**

 Next.js 12.0.5

**What version of Node.js are you using?**

For example: v16.13.1

**What browser are you using?**

For example: Chrome

**What operating system are you using?**

For example: macOS

**Reproduction URL**

N/A

**Describe your issue**

Style changes doesn't gets updated after i change the classes and save the file, literally no way to see the changes. Even happens with 3.0.0-alpha-2

**Comment by adamwathan** [comment]
Please provide a reproduction, we use Next for lots of things and don’t have this issue.

**Reply by adamwathan** [reply]
Looks like @RobinMalfait was actually able to reproduce this, appears to be specific to Next.js 12.0.5. Can pin to 12.0.4 and it should work 👍🏻 @timneutkens from the Vercel team is looking into it on their end.

**Reply by timneutkens** [reply]
I've opened up a PR here: https://github.com/vercel/next.js/pull/32130

**Comment by afkcodes** [comment]
Thanks @adamwathan for confirming this, will move to 12.0.4 now and also please let me know the reproducible repo is still needed or not.
